packages/connect-form/src/components/connect-form.tsx

```diff
@@ -48,10 +48,18 @@ const descriptionStyles = css({
   marginTop: spacing[2],
 });
 
+const scrollbarWidth = spacing[2];
 const formContentContainerStyles = css({
   padding: spacing[4],
-  overflow: 'scroll',
+  paddingRight: spacing[4] - scrollbarWidth,
+  overflowY: 'scroll',
   position: 'relative',
+  scrollPadding: 0,
+  scrollbarGutter: 'unset',
+  scrollbarWidth: 'none',
+  '::-webkit-scrollbar': {
+    width: scrollbarWidth,
+  },
```

Comment:
We now have scrollbars with two different widths on the screen at the same time which looks a bit odd, I think we might want to either set this value globally, or (as I mentioned in the comment above) not change it alltogether. What do you think?

<img width="195" alt="image" src="https://user-images.githubusercontent.com/5036933/150785516-1f8f87dc-e1b5-411c-9625-68bc5d1c8057.png">

Reply:
I'd love the `overflow-y: auto` if it works!

Reply:
if not we could just have an <Scroll> component or something shared in compass-component as variable?

Reply:
I mean, depends on what you mean by "works" 😅

`auto` will only show the scrollbar (and shift the content with additional) when the content overflow the container. Content can overflow (and scrollbar can show up) in like a million places, so I'm not even sure what exactly we can put into a shareable component here if we consider this behavior a problem. A `Box` component that we would need to wrap around almost everything seems like a bit too much work for something like this

Reply:
Ok `auto` worked, I think that is good enough!